### PR TITLE
Added autofocus on search input on plugin hub page

### LINKED ISSUE #345: Give focus on load to search bar on plugin hub page
When opening the [plugin hub page](https://runelite.net/plugin-hub) it would be nice if the on load focus was set to the search bar, allowing you to search for a plugin without having to first click in the search bar text field or tabbing 14 times
image
Pros:
- Efficiency

Cons:
- Literally none

### PR DESCRIPTION
**Pull request by arthurtyukayev**
Closes #345

### CODE REVIEW

**Comment by deathbeam** [comment]
https://stackoverflow.com/a/40235334/4915340 can't you simply do this? I tested it locally and it works

File: src/routes/plugin-hub.js [hunk 1]
```diff
@@ -45,6 +45,7 @@ const PluginHub = ({ externalPlugins, pluginFilter, setPluginFilter }) => (
         <SearchBar
           value={pluginFilter.name}
           onInput={e => handleChange(e, setPluginFilter)}
+          ref={input => input && input.focus()}
```
Comment (deathbeam) [inline src/routes/plugin-hub.js:48]: I would just put this directly into search bar and remove the forwardRef. I think its fine to have all search bars auto focused

Comment (deathbeam) [reply src/routes/plugin-hub.js:48]: @arthurtyukayev ^

Comment (arthurtyukayev) [reply src/routes/plugin-hub.js]: Done @deathbeam

**Pull request merged**